User_class.rb

```diff
@@ -0,0 +1,28 @@
+# freeze string literal: true
+
+# User view model.
+class User
+  attr_accessor :first_Name, :last_Name, :email
```

Comment:
The tutorial said to create it this way then went on to say it would be used in the next chapter. I don't know the reasoning for it but I'm trusting the tutorial has a plan. Likewise I assume classes are typically added to the root directory of an application.